numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -104,6 +104,10 @@ def simple_lmem(A, B, dty):
         B[i] = C[i]
 
 
+parent_dir = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Maybe use `pathlib.Path` here and the `.parent` property instead?

Reply:
Yeah, that would be better,

Reply:
Now used in the declaration of `test_data_dir` in `numba.tests.support`.